pkg/operator/ceph/cluster/cluster.go

```diff
@@ -67,6 +70,16 @@ type cluster struct {
 	monitoringActivated  bool
 }
 
+func getCrushRootFromSpec(c *cephv1.ClusterSpec) string {
+	if c.Storage.Config == nil {
+		return ""
+	}
+	if v, ok := c.Storage.Config[crushRootConfigKey]; ok {
+		return v
+	}
+	return ""
```

Comment:
Why not returning `default` here? So that we will always populate it and we don't need to check if empty in other portion of the code?